metarecord/migrations/0001_squashed_0048_replace_pg_jsonfield_with_django_jsonfield.py

```diff
@@ -0,0 +1,1757 @@
+# Generated by Django 2.2.16 on 2021-01-15 06:25
```

Comment:
maybe the name of this file should be just `0001_squashed_0048.py` since `replace_pg_jsonfield_with_django_jsonfield` is just the last part, but there are so many others too.